internal/tiltfile/telemetry/telemetry.go

```diff
@@ -0,0 +1,80 @@
+package telemetry
+
+import (
+	"fmt"
+
+	"go.starlark.net/starlark"
+
+	"github.com/windmilleng/tilt/internal/tiltfile/starkit"
+	"github.com/windmilleng/tilt/pkg/model"
+)
+
+type Settings struct {
+	Cmd model.Cmd
+}
+
+type Extension struct {
+}
+
+func NewExtension() Extension {
+	return Extension{}
+}
+
+func (e Extension) NewState() interface{} {
+	return Settings{
+		Cmd: model.Cmd{},
+	}
+}
+
+func (Extension) OnStart(env *starkit.Environment) error {
+	return env.AddBuiltin("experimental_telemetry_cmd", setTelemetryCmd)
+}
+
+func setTelemetryCmd(thread *starlark.Thread, fn *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
+	var cmd string
+	err := starkit.UnpackArgs(thread, fn.Name(), args, kwargs, "cmd", &cmd)
+	if err != nil {
+		return starlark.None, err
+	}
+
+	if len(cmd) == 0 {
+		return starlark.None, fmt.Errorf("cmd cannot be empty")
+	}
+
+	var innerErr error
+	err = starkit.SetState(thread, func(settings Settings) Settings {
+		if len(settings.Cmd.Argv) > 0 {
+			innerErr = fmt.Errorf("%v called multiple times; already set to %v", fn.Name(), settings.Cmd)
+			return settings
+		}
+
+		settings.Cmd = model.ToShellCmd(cmd)
```

Comment:
should the Telemetry command also inherit the working directory of the Tiltfile? That would be easy to add and dovetail nicely with L's work. Then you could load() a remote github repo with your Tiltfile + telemetry script, which would be useful for orgs with multiple repos.